_posts/2020-10-05-how-to-write-a-new-foreman-ansible-module-in-20-lines-of-code.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: How to write a new Foreman Ansible module in 20 lines of code
+date: 2020-10-05 11:29:55
+author: Evgeni Golov
+tags:
+- foreman
+---
+
+Some time ago, we were asked ([twice](https://bugzilla.redhat.com/show_bug.cgi?id=1856401)) to [add a module for managing HTTP proxies in Foreman](https://github.com/theforeman/foreman-ansible-modules/issues/867) to the Foreman Ansible Modules collection. Given the limited scope of the module, it's a perfect fit for a guide how to write new modules utilizing the abstraction we have developed in Foreman Ansible Modules!
+
+<!--more-->
```

Comment:
??

Reply:
this is a marker for Jekyll to add the first part on the main https://theforeman.org/blog/ page and the rest after the reader opens the real post.